packages/ai/src/PromptInput.ts

```diff
@@ -0,0 +1,285 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
+import Input, { InputEventDetail } from "@ui5/webcomponents/dist/Input.js";
+import Label from "@ui5/webcomponents/dist/Label.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+import {
+	isEnter,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import {
+	PROMPT_INPUT_CHARACTERS_LEFT,
+	PROMPT_INPUT_CHARACTERS_EXCEEDED,
+} from "./generated/i18n/i18n-defaults.js";
+
+import PromptInputTemplate from "./generated/templates/PromptInputTemplate.lit.js";
+
+// Styles
+import PromptInputCss from "./generated/themes/PromptInput.css.js";
+
+type ExceededText = {
+	exceededText?: string;
+	leftCharactersCount?: number;
+	calcedMaxLength?: number;
+};
+
+/**
+ * @class
+ * ### Overview
+ *
+ * The `ui5-ai-prompt-input` component allows the user to write custom instructions in natural language, so that AI is guided to generate content tailored to user needs.
+ *
+ * **Note:** The web component is in an experimental state
+ *
+ * ### ES6 Module Import
+ *
+ * `import "@ui5/webcomponents-ai/dist/PromptInput.js
+ * @class
+ * @constructor
+ * @public
+ * @extends UI5Element
+ */
+@customElement({
+	tag: "ui5-ai-prompt-input",
+	renderer: litRender,
+	styles: PromptInputCss,
+	template: PromptInputTemplate,
+	dependencies: [
+		Input,
+		Label,
+		Button,
+	],
+})
+
+/**
+ * Fired when the input operation has finished by pressing Enter
+ * or AI button is clicked.
+ *
+ * @since 2.0.0-rc.1
+ * @public
+ */
+@event("submit")
```

Comment:
I don't understand the reasoning behind having both `submit` and `change`. Submit is fired on click/enter, while change is fired on enter/focusout. What am i supposed to bind as an app developer? I would prefer to have only input/change events.